storage/tests/unit/test_blob.py

```diff
@@ -1187,7 +1190,6 @@ def _do_resumable_helper(self, use_size=False, num_retries=None):
         client = mock.Mock(_http=transport, spec=['_http'])
         stream = io.BytesIO(data)
         content_type = u'text/html'
-        predefined_acl = 'private'
         response = blob._do_resumable_upload(
             client, stream, content_type, size, num_retries, None)
```

Comment:
This should be passing through the `predefined_acl` argument, and checking that it is passed through in the assertions.